ros2_ws/src/vision/path_detection/path_detection/obstacles.py

```diff
@@ -88,13 +86,13 @@ def state_callback(self, new_state):
         self.state = new_state.data
 
     def get_c(self, i, scan):
-        return -(190 * (452-math.cos(i * scan.angle_increment)) - 452 * (190-math.sin(i * scan.angle_increment)))
+        return -(190 * (452-math.cos(i * scan.angle_increment)) - 452 * (190-math.sin(i * scan.angle_increment))) # is this center of obstacle or pothole??
```

Comment:
probably should figure out the answer to this question. let me know if you want help.

Reply:
only used for potholes. put a comment explaining that we didn't update it